### PR TITLE
Fix the app

### PR DESCRIPTION
**Pull request by huzaif-plivo**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,63 +1,56 @@
...
-  let reqPost = https.request(postOptions, function(response) {
-    console.log("statusCode: ", response.statusCode);
-    response.on('data', function(d) {
-      console.info('POST result:\n');
-      process.stdout.write(d);
-      console.info('\n\nPOST completed');
-      res.send(d);
+  
+    jsonObject = JSON.stringify({
+        "phoneMeNumber"     : req.fields.src,
```
Comment (samuellawerentz) [inline server/index.js]: Can we remove extra spaces after "phoneMeNumber"?
And similarly in other objects as well.

File: server/index.js [hunk 2]
```diff
@@ -1,63 +1,56 @@
...
+  
+    // prepare the header
+    let postHeaders = {
+      'Content-Type' : 'application/json',
+      'Authorization': 'Basic ' + new Buffer.from(process.env.PLIVO_AUTH_ID + ':' + process.env.PLIVO_AUTH_TOKEN).toString('base64')
+    };
+  
+    // set the post options
+    let postOptions = {
+      port   : 443,
```
Comment (samuellawerentz) [inline server/index.js]: Remove extra spaces

**Review by samuellawerentz — APPROVED** [review]
Looks Good!
There are few linting errors, please address them.

File: server/index.js (hunk 1 above)
Comment (huzaif-plivo) [reply server/index.js]: Done @samuellawerentz

File: server/index.js (hunk 2 above)
Comment (huzaif-plivo) [reply server/index.js]: Done @samuellawerentz

**Comment by huzaif-plivo** [comment]
> Looks Good! There are few linting errors, please address them.

@samuellawerentz added .eslintrc to solve linter errors.

**Review by samuellawerentz — APPROVED** [review]

**Pull request merged**